server/chat-plugins/helptickets.ts

```diff
@@ -352,6 +352,19 @@ export class HelpTicket extends Rooms.RoomGame {
 		// @ts-ignore
 		this.playerTable = null;
 	}
+	onChatMessage(message: string, user: User) {
+		// in case they namechange
```

Comment:
Does it matter if they namechange?

Reply:
The comment refers to the `room.auth.has` check, where yes, it's ensuring that just the ticketer's replays get uploaded.

Reply:
In other words, if the ticket creator changes names, the replays they post shouldn't be uploaded? Why?

This is also the reverse of our comment standards. This comment just says what the code does, which anyone who can read JS already knows. What it doesn't explain is why it does it, which is what comments are for.

Reply:
> In other words, if the ticket creator changes names, the replays they post shouldn't be uploaded?
Vice versa. The creator's posted replays should always be uploaded, regardless of the name - that's why it checks room.auth rather than a direct userid check.